webpack.config.babel.js

```diff
@@ -122,6 +127,7 @@ const PROD_CONFIG = {
   SMS_DELIVERY_URL: 'https://telephony.onfido.com',
   PUBLIC_PATH: `https://assets.onfido.com/web-sdk-releases/${packageJson.version}/`,
   USER_CONSENT_URL: 'https://assets.onfido.com/consent/user_consent.html',
+  OLD_AUTH: 'https://edge.api.onfido.com/v3',
```

Comment:
It's a bit vague to introduce a new `OLD_AUTH` here, can you explain the context?

Reply:
not 100% if this is what Carla intended here, but I believe this URL will change to a new one, will check

Reply:
`OLD_AUTH` was referring to an old url we were using to test the earlier stages of this implementation- it went by without an update to the name, which is due 😓  my bad. This URL is for our own production backend for auth, so we should keep using it.

Reply:
Maybe `LEGACY_AUTH` might be a better name for this?

Reply:
For now I'll change it to `AUTH_URL` as this one is the production one we actually use, if you agree?